taxi/forms.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class CustomUserCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "license_number",
+            "first_name",
+            "last_name"
+        )
+
```

Comment:
read again in README file in checklist it is indicated there: ```2. Validate license_number while creating and updating users.```
So, you need implement method clean_license_number also to CustomUserCreationForm